### ISSUE TITLE
Getting Started is missing sample app.html output

### ISSUE THREAD

**Issue by canadaduane** [jspm/jspm.org#32]
I believe there is meant to be an example output after the line,

```
The generated app.html now includes both our local code mapping as well as the lit mapping:
```

at

https://github.com/jspm/jspm.org/blob/main/pages/getting-started.md?plain=1#L110

**Comment by guybedford** [comment]
Right, some kind of sample would be nice to include there. It might have been partially removed due to version sync issues, I'm not sure.
